assets/src/lightbox/web-stories-lightbox.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Handles the 'amp-story-player' lightbox.
+ *
+ * Displays the story that user interacted with in a lightbox.
+ */
+class Lightbox {
```

Comment:
What is the reason you are using a class here? All other code use functional / hook pattern. Do you think it is a problem that code is different @swissspidy ?

Reply:
Good question.

It's certainly out of the ordinary, but seems like a good fit in this case to wrap every found element in a lightbox.